zwave_me_server/config.yaml

```diff
@@ -1,34 +1,42 @@
 name: Z-Wave.Me Add-on
-startup: services
-version: 4.1.0
+version: 4.1.2
 slug: zwave_me_server
 description: 'Z-Wave.Me Z-Way add-on for Home Assistant'
+startup: services  # Determines when the add-on should start
 arch:
   - amd64
   - aarch64
-  - armhf
-url: https://github.com/z-wave-me/ha-z-wave-me-addon/
+  - armhf  # Architectures the add-on can run on
+init: false
```

Comment:
why?

Reply:
Because now the z-way startup process is not controlled by the built-in docker system, but uses s6-overlay, which already controls the launch of services inside the addon